### PR TITLE
Add html file

### PR DESCRIPTION
**Pull request by mark-drastrup**
Hi everyone!

I have added a html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Git demo</title>
```
Comment (mark-drastrup) [inline index.html:6]: I don't like this title

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (a-yasutommy) [inline index.html:5]: I don't like it. Do it again.

Comment (hiroho-t) [inline index.html:5]: yeeeeei

**Comment by mark-drastrup** [comment]
This is some good coding!

**Review by a-yasutommy — COMMENTED** [review]
@mark-drastrup You have to correct some part!!

File: index.html (hunk 1 above)
Comment (hiroho-t) [inline index.html:6]: @nozomi-I hellooo

**Comment by nozomi-I** [comment]
aaaaa @mark-drastrup

**Review by a-yasutommy — APPROVED** [review]

**Review by hiroho-t — APPROVED** [review]

**Review by nozomi-I — APPROVED** [review]
OK!

**Pull request merged**